cluster/manifests/node-monitor/daemonset.yaml

```diff
@@ -73,6 +73,8 @@ spec:
               protocol: TCP
         - image: registry.opensource.zalan.do/teapot/prometheus-node-exporter:v1.2.2
           args:
+            - --collector.ethtool
```

Comment:
Please test that this doesn't cause memory usage issues due to the veth* interfaces.